CHANGELOG.mkd

```diff
@@ -3,10 +3,20 @@ CHANGELOG
 
 Unreleased
 ----
-- Add exec environment source type. The exec source type allows for the implementation of external environment sources [#1042](https://github.com/puppetlabs/r10k/pull/1042)
+
+
+3.5.0
+-----
+
+- Add exec environment source type. The exec source type allows for the
+  implementation of external environment sources
+  [#1042](https://github.com/puppetlabs/r10k/pull/1042).
```

Comment:
this links to a PR too

Reply:
Ha, I totally read it as an issue.

Reply:
Looking into it, I'm not sure what to do with that one since there's no issue or ticket with it but afaict it does implement a new feature. I think I'd leave the link to the PR in, not because I think folks should go see the back and forth about the code review, or go read the code, but because it's the best place to find a high level understanding of the feature...  What do you think?

Reply:
hmm yeah, I get that.. I don't know, I hate changelogs lol

Reply:
Yeah I think it's less important what is linked to, than that the link takes you somewhere to get more detail.